tests/conftest.py

```diff
@@ -81,3 +88,14 @@ def validator(data_json):
         return data_json
 
     return validator
+
+
+@pytest.fixture
```

Comment:
Woo, this looks great! :tada:

Reply:
Yeah I had some weird side effects in other tests, but only when I would run the whole suite. This should fix that up.